libshaderc/include/shaderc.h

```diff
@@ -105,6 +111,11 @@ void shaderc_compile_options_release(shaderc_compile_options_t options);
 void shaderc_compile_options_add_macro_definition(
     shaderc_compile_options_t options, const char* name, const char* value);
 
+// Set flags for the rules on which warnings or errors will be issued. This can
+// either be an individual flag, or a set of flags or'd together
```

Comment:
If `rules` can be a combo of enum values, then its declared type can't be the enum.